.github/workflows/build-ci.yml

```diff
@@ -59,6 +55,13 @@ jobs:
 
         steps:
             -   uses: actions/checkout@v2
+            -   name: Create MongoDB Replica Set
+                run: |
+                    docker run --name mongodb -p 27017:27017 -e MONGO_INITDB_DATABASE=unittest --detach mongo:${{ matrix.mongodb }} mongod --replSet rs
```

Comment:
Remember to use `--setParameter transactionLifetimeLimitSeconds=5` (or a lower value) to minimize deadlock time in CI.

Reply:
Done.